docs/docs/manual/exporting.md

```diff
@@ -6,34 +6,117 @@ sidebar_label: Exporting
 
 ## Overview
 
+Once your data is cleaned, you will need to get it out of OpenRefine and into the system of your choice. OpenRefine outputs a number of file formats, can upload your data directly into Google Sheets, and can create or update statements on Wikidata.
 
-## Export data
+You can also [export your full project data](#export-a-project) so that it can be opened by someone else using OpenRefine (or yourself, on another computer).
 
-Note you will only export data in the current view - that is, with current filters and facets applied. 
+## Export data
 
+Many of the following options only export data in the current view - that is, with current filters and facets applied. Some will give you the choice to export your entire dataset or just your current view.
 
+To export from a project, click the <span class="menuItems">Export</span> dropdown button at the top right corner and pick the format you want. You options are:
 
-*   TSV/CSV
-*   HTML table
+*   Tab-separated value (TSV) or Comma-separated value (CSV)
+*   HTML-formatted table
 *   Excel (XLS or XLSX)
 *   ODF spreadsheet
-*   Google Sheets \
+*   Upload to Google Sheets (requires [Google account authorization](starting#google-sheet-from-drive))
+*   [Custom tabular exporter](#custom-tabular-exporter)
+*   [SQL statement exporter](#sql-statement-exporter)
```

Comment:
There's a mismatch between this link (#sql-statement-exporter) and the anchor further down (#sql-exporter) meaning that the link doesn't work